integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/testsupport/dialect/Elasticsearch60TestDialect.java

```diff
@@ -50,4 +50,10 @@ public void setTemplatePattern(JsonObject object, String pattern) {
 		array.add( pattern );
 		object.add( "index_patterns", array );
 	}
+
+	@Override
+	public boolean supportLargeBigIntegerProjections() {
```

Comment:
Does it work with ES 6.5? If so, I could upgrade our testing infrastructure to use ES 6.5 on AWS (I should do it anyway), and we could remove this commit entirely. We'll just avoid testing regularly on ES 6.4.

Reply:
With ES 6.5 it works. Let's drop this commit. Of course full CI would not work until we upgrade our testing infrastructure.